graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/internal/DataFetcherInvoker.kt

```diff
@@ -239,8 +240,15 @@ class DataFetcherInvoker(
             } else {
                 InputObjectMapper.mapToJavaObject(parameterValue as Map<String, *>, targetType)
             }
-        } else if (parameter.type.isEnum && parameterValue !== null) {
-            (parameter.type.enumConstants as Array<Enum<*>>).find { it.name == parameterValue }
+        } else if ((parameter.type.isEnum || collectionType?.isEnum ?: false) && parameterValue != null) {
```

Comment:
What does this mean? `collectionType?.isEnum ?: false` Should this just be `collectionType?.isEnum`

Reply:
It should just be `collectionType?.isEnum == true`, the Kotlin compiler doesn't like just `collectionType?.isEnum` 🤷

Reply:
Got it, I see.